<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updatet</class>
 <widget class="QDialog" name="updatet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter source</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter destination</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Old Transportation</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>New Transportation</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Cost</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
